### DISCUSSION TITLE
Promoting a collaborator to Owner (or similar)

### DISCUSSION THREAD

**Discussion by perdrix52** [community/community#22499] [New to GitHub]
<p>To make sure we’re talking about the same thing - I’m faced with a problem on the LucCoiffier/DSS repository (which isn’t a “Organization” type repository).</p>
<p>The original Owner (Luc Coiffier) is not really active these days, so I asked him to promote me to Administrator/Owner level.  However the Collaborators page doesn’t appear to offer this capability.</p>
<p>I suppose I could fork this project to perdrix52/DSS but and then tell the world that LucCoiffier/DSS is defunct, but I’d much rather <strong>not</strong> do that.</p>
<p>What’s the trick please - I assume this must be possible?</p>
<p>Thanks, David Partridge</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>The owner of a repository is capable of  <strong>transferring ownership</strong> of a repository by visiting the  <strong>Settings</strong> of the repository they want to transfer ownership of. Once there, the owner must scroll down to the “Danger Zone” and click the <em>Transfer</em> button and type in the current repository name and the new owner’s name.</p>
<p>For all information about transferring ownership of a repository, you can visit the <a href="https://help.github.com/en/articles/transferring-a-repository" title="Transferring Repository Ownership" rel="nofollow noopener">GitHub Help</a> article regarding this topic! Hope this helps.</p>

**Comment by ghost** [comment]
<p>Unfortunately, this doesn’t appear to be working for me? It’s been many hours, and the repos aren’t transferring to the new owner. Is there a step that article may be missing?</p>
